CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - [cosmos-hive-auth-provider][hardening] Improve logs (#227)
+- [cosmos-hive-auth-provider][feature] Add OAuth2 cache (#225)
```

Comment:
Added in commit https://github.com/telefonicaid/fiware-cosmos/pull/230/commits/ed4debe7e4266e06a2e74faeb8fb089d2572dd11